app/editor/src/features/admin/av-overviews/OverviewSubscribers.tsx

```diff
@@ -0,0 +1,64 @@
+import { useFormikContext } from 'formik';
+import React from 'react';
+import { useUsers } from 'store/hooks/admin';
+import { FlexboxTable, ITableInternal, ITablePage, ITableSort, IUserModel } from 'tno-core';
+
+import { subscriberColumns } from './constants';
+import { IAVOverviewTemplateForm } from './interfaces';
+import { ListFilter } from './ListFilter';
+
+/**
+ * The page used to view and edit reports.
+ * @returns Component.
+ */
+export const OverviewSubscribers: React.FC = () => {
+  const { values, setFieldValue } = useFormikContext<IAVOverviewTemplateForm>();
+  const [{ users }, { findUsers }] = useUsers();
+
+  React.useEffect(() => {
+    async function getUsers() {
+      await findUsers({ page: 1, quantity: users.quantity });
```

Comment:
Why are you wrapping this function?  Why not just call it?

Reply:
Yeah, that is what I tried to do initially, however, a warning shows and recommends to use the current implementation with it, but I'm open if there is a better way for this.

Reply:
Just do `findUsers({page: 1, quantity: users.quantity }).catch(() => {});`

Reply:
Updated, thanks